src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs

```diff
@@ -96,10 +103,9 @@ public class BlockchainServiceWeb3 : JsonRpcService
                 Logger.LogWarning($"txs {txs}");
                 foreach (var tx in txs)
                 {
-                    Logger.LogWarning($"tx {tx.Hash} {tx.GasUsed} {tx.Status} {tx.IndexInBlock}");
+                    Logger.LogWarning($"tx {tx.Hash.ToHex()} {tx.GasUsed} {tx.Status} {tx.IndexInBlock}");
```

Comment:
We need to check if tx is not empty here
```suggestion
                    if (tx is null) 
                         continue;
                    Logger.LogWarning($"tx {tx.Hash.ToHex()} {tx.GasUsed} {tx.Status} {tx.IndexInBlock}");
```